### PR TITLE
Update page's organisation mentions to the new name

### PR DESCRIPTION
**Pull request by uncojohnco**
Change page mentions of org. name "gastown gang" to "cambie collective".

## TODO:
- [x]  Should twitter handle https://twitter.com/gastowngang be changed to https://twitter.com/cambiecollective?

---

Resolves: #26 
Related: #14

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ public
 .intermediate-representation/
 .cache/
 yarn.lock
+.idea
```
Comment (keywordnew) [inline .gitignore]: Confirming it's for the Jetbrains IDE?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "landing-page",
-  "description": "the front-facing page for Gastown Gang",
-  "version": "0.1.0",
+  "description": "the front-facing page for Cambie Collective",
+  "version": "0.1.1",
```
Comment (keywordnew) [inline package.json:4]: 👍🏽

File: src/components/Footer.js [hunk 3]
```diff
@@ -5,16 +5,16 @@ const Footer = (props) => (
...
-                <li><a href="https://github.com/gastowngang/gastowngang.github.io" className="button">Join us!</a></li>
+                <li><a href="https://github.com/cambiecollective/cambiecollective.github.io" className="button">Join us!</a></li>
             </ul>
         </section>
         <section>
             <h2>Find us on the internet</h2>
             <ul className="icons">
-                <li><a href="https://twitter.com/gastowngang" className="icon fa-twitter alt"><span className="label">Twitter</span></a></li>
-                <li><a href="https://github.com/gastowngang/gastowngang.github.io" className="icon fa-github alt"><span className="label">GitHub</span></a></li>
+                <li><a href="https://twitter.com/cambiecollective" className="icon fa-twitter alt"><span className="label">Twitter</span></a></li>
```
Comment (keywordnew) [inline src/components/Footer.js]: As the very observant todo pointed out in the PR body, the twitter did need updating :)

For now it's `cambiecollectiv` without the "e". Thanks to Twitter's handle length limits. The link here can be changed to reflect that.

**Review by keywordnew — COMMENTED** [review]
Thank you for opening the PR! ✨✨✨

LGTM once the comments are resolved.

File: src/components/Footer.js (hunk 3 above)
Comment (Qard) [reply src/components/Footer.js]: TIL: My full name on Twitter is _exactly_ the max length. 😅

**Comment by keywordnew** [comment]
Thank you for resolving the comments @oikake! 💯

Merging PR 💁🏽‍♀️

**Comment by keywordnew** [comment]
@oikake Long overdue greeting, "Welcome to the Collective!"

**Pull request merged**